hash/nes.xml

```diff
@@ -35902,6 +35902,23 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="spyvsspy2p" cloneof="nangsspy">
+		<description>Spy vs. Spy: The Island Caper (USA, Prototype)</description>
+		<year>1988</year>
+		<publisher>Kemco</publisher>
+		<part name="cart" interface="nes_cart">
+			<feature name="slot" value="uxrom" />
+			<feature name="pcb" value="HVC-UNROM" />
+			<feature name="mirroring" value="vertical" />
+			<dataarea name="prg" size="131072">
+				<rom name="spy vs spy - the island caper" size="131072" crc="c2bbfc2b" sha1="00e969c7430c02100584ea7caa0cb4564975bb87" offset="00000" />
```

Comment:
Please keep the size and offset together when adding ROMs.  However, I think @0kmg was leaning towards not including offsets at all in this software list when there’s a single ROM in a region, as the bulk of NES games tend to use a single ROM chip for each data area.

Reply:
Oh sorry, I saw the offset posted at the end in other items, so I did the same here. If you want me to delete it, lemme know and I'll get to it later today.

Reply:
Seems that the title for this game on Hidden Palace has a "II" in it:
https://hiddenpalace.org/Spy_vs._Spy_II:_The_Island_Caper_(US_prototype)

Reply:
Although, I see that the screenshot of the title screen there lacks the "II".

Reply:
Yeah, it's not in the title screen, and since there's no box art (no release) and none of the other known releases have the number on the title screen or cover, I decided to omit it. :)

https://www.mobygames.com/game/7698/spy-vs-spy-the-island-caper/

Reply:
This flyer implies that the NES box was going to have a II in it but yeah, not on the title screen:
![image](https://github.com/mamedev/mame/assets/47050710/d28822c1-de43-4809-b798-1dea3809d6bc)

Reply:
> Oh sorry, I saw the offset posted at the end in other items, so I did the same here. If you want me to delete it, lemme know and I'll get to it later today.

Yeah, just remove the offset for this one – it isn’t really necessary for NES cartridges.